javascript/node/selenium-webdriver/test/http/http_test.js

```diff
@@ -100,6 +102,31 @@ describe('HttpClient', function() {
     }
   });
 
+  server.setConnectHandler(function(req, sock, head) {
+    if (req.method == 'CONNECT' && req.url == "another.server.com") {
+      // sock.on('data', function(str) { console.log(str.toString()); });
+      sock.write('HTTP/1.1 200 Connection Established\r\n\r\n');
+      let keydata = fs.readFileSync("test/http/server-key.pem");
```

Comment:
Please generate self signed server keys in the before function of the test, everyone who would run the test would need to manually figure out what to do.